src/utils/sources-tree.js

```diff
@@ -224,13 +225,27 @@ function addToTree(tree: any, source: TmpSource) {
 }
 
 /**
+ * @memberof utils/sources-tree
+ * @static
+ */
+function isExactUrlMatch(pathPart:string, debuggeeUrl:string) {
+  // compare to hostname with an optional 'www.' prefix
+  const { host } = parse(debuggeeUrl);
+  if (!host) {
+    return false;
+  }
+  const baseUrl = host.replace(/^www\./, "");
+  return pathPart === baseUrl || pathPart === `www.${baseUrl}`;
```

Comment:
this seems slightly more complicated than it needs to be. Could we also replace the `www` for the pathPart?

```js
function isExactUrlMatch(pathPart:string, debuggeeUrl:string) {
  function getHost(url) {
    const { host } = parse(debuggeeUrl);
    if (!host) {
      return ""
    }
    return host.replace(/^www\./, "");
  }

  return getHost(pathPart) == getHost(debuggeeUrl)
}
```